src/travis2docker/travis2docker.py

```diff
@@ -131,6 +130,9 @@ def _compute_env(data, _):
                 continue
             env_globals += " " + env_global
         env_globals = env_globals.strip()
+        psql_version = self.yml.get('addons', {}).get('postgresql')
+        if psql_version:
+            env_globals += ' PSQL_VERSION="%s"' % psql_version
```

Comment:
Could you add a test case to cover this line?
Add a new `.travis.yml` file using this paremeter.

Reply:
ok, Got it

Reply:
@moylop260 
Done

Reply:
Did you run a `tox` or `python setup.py test`?

(Because travis is red)

Reply:
I ran `python setup.py test`, although it seemed to test nothing.

I just tried using tox and it failed, I'll check.